libraries/iio/iio.c

```diff
@@ -0,0 +1,843 @@
+/***************************************************************************//**
```

Comment:
Isn't this copy realized in a previous pull request?
https://github.com/analogdevicesinc/no-OS/pull/657/commits/d30c3e72833be2882dc9a2d70a41b7ce30b7c544